docs/README.md

```diff
@@ -1,8 +1,10 @@
 # Open Prices
```

Comment:
Maybe you wanted to edit the root README instead ? (and we could delete this docs/README ?)